src/test/java/com/crowdin/util/PropertyUtilTest.java

```diff
@@ -1,76 +1,45 @@
 package com.crowdin.util;
-
-import com.crowdin.client.CrowdinProperties;
-import com.crowdin.client.CrowdinPropertiesLoader;
-import com.crowdin.client.FileBean;
-import com.crowdin.client.FileBeanBuilder;
+import com.intellij.testFramework.fixtures.BasePlatformTestCase;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
-import java.io.FileInputStream;
-import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
 import java.util.stream.Stream;
 
-import static org.junit.Assert.assertEquals;
 import static org.junit.jupiter.params.provider.Arguments.arguments;
 
-public class PropertyUtilTest {
+public class PropertyUtilTest extends BasePlatformTestCase {
+
+    @Override
+    protected String getTestDataPath() {
+        return "src/test/testData";
+    }
+
+    @BeforeEach
+    public void setup() throws Exception {
+        super.setUp();
+    }
+
+    @AfterEach
+    public void teardown() throws Exception {
+        super.tearDown();
+    }
 
     @ParameterizedTest
     @MethodSource
-    public void testGetSources(Properties properties, List<FileBean> expected) {
-        CrowdinProperties result = CrowdinPropertiesLoader.load(properties);
-        assertEquals("Properties: " + properties, expected, result.getFiles());
+    public void testGetSources(String project, String expected, String key) {
```

Comment:
It's a duplicated test name. The names should be unique and describe what is being tested in this test